### ISSUE TITLE
A11y keyboard control Tab component TabList focus issue

### ISSUE THREAD

**Issue by DaveBest99** [angular/components#3420]
#### Bug, feature request, or proposal:
Bug: Screen reader focus issue.

#### What is the expected behavior?
Pressing the Tab key to move focus into the TabList, should place the focus on the active Tab Element. Only the active Tab element should be in the focus order. The active Tab element can be identified by the "tabindex=0" or the "aria-activedescendent=id". All other TabList Tab elements are removed from the focus order with "tabindex=-1".

#### What is the current behavior?
Currently the screen reader focus is on the first Tab element in the TabList instead of the selected Tab element.

#### Which versions of Angular, Material, OS, browsers are affected?
Testing was performed with Windows10 using the JAWS17 screen reader with Firefox/IE11/Chrome browsers.

#### Is there anything else we should know?
The TabList does not indicate correct focus Tab element.
JAWS always moves focus to the first TabList item and speaks the text label "Tab1", even if it is not the currently selected item. However, the actual focus appears to be on the List Item that the JAWS focus was last on. That is, each time the user reenters the TabList, JAWS reads the first item text but the actual focus is on the item that was last viewed. If "Tab1" item is selected, but upon reentering the TabList the user presses the Right Arrow key to "Tab2", and then leaves the TabList; upon reentry into the TabList the focus is on "Tab2" instead of the "Tab1" item with "tabindex=0".
For example, upon reentering the TabList that contains 3 Tab elements (Overview, API Reference, Examples) where "API Reference" is selected but the "Examples" Tab was the last user focus point previously, JAWS speaks "Overview", but the focus is on "Examples". Focus should always land on the TabList Tab element with "tabindex=0" or "aria-activedescendent=id"; Which in this example is "API References". Note, the braille display shows "Overview" confirming JAWS speech output.
